lldb/packages/Python/lldbsuite/test/lang/swift/runtime_failure_recognizer/Makefile

```diff
@@ -0,0 +1,6 @@
+LEVEL = ../../../make
```

Comment:
This should be redundant?

Reply:
I tried to remove it, with the include (few lines below) but I keep getting Unexpected Failures.